pom.xml

```diff
@@ -21,205 +21,235 @@
-			<url>https://maven.forgerock.org:443/repo/community</url>
-		</repository>
-		<snapshotRepository>
-			<id>maven.forgerock.org-community-snapshots</id>
-			<name>maven.forgerock.org-snapshots</name>
-			<url>https://maven.forgerock.org:443/repo/community</url>
-		</snapshotRepository>
-	</distributionManagement>
-
-	<repositories>
-	</repositories>
-
-	<licenses>
-		<license>
-			<name>Apache 2.0</name>
-			<url>http://www.apache.org/licenses/LICENSE-2.0</url>
-		</license>
-	</licenses>
-
-	<developers>
-		<developer>
-			<name>Quentin Castel</name>
-		</developer>
-	</developers>
-
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.8.1</version>
-				<inherited>true</inherited>
-				<configuration>
-					<source>${maven.compiler.source}</source>
-					<target>${maven.compiler.target}</target>
-				</configuration>
-			</plugin>
-
-			<plugin>
-				<artifactId>maven-release-plugin</artifactId>
-				<version>2.5.3</version>
-				<configuration>
-					<scmCommentPrefix>[ci skip]</scmCommentPrefix>
-				</configuration>
-			</plugin>
-			<plugin>
-				<groupId>com.mycila</groupId>
-				<artifactId>license-maven-plugin</artifactId>
-				<version>3.0</version>
-				<configuration>
-					<header>file://${maven.multiModuleProjectDirectory}/legal/LICENSE.txt</header>
-					<includes>
-						<include>**/*.java</include>
-						<include>pom.xml</include>
-					</includes>
-					<excludes>
-						<exclude>**/README</exclude>
-						<exclude>**/*sh</exclude>
-						<exclude>**/*txt</exclude>
-						<exclude>settings.xml</exclude>
-						<exclude>src/test/resources/**</exclude>
-						<exclude>src/main/resources/**</exclude>
-					</excludes>
-				</configuration>
-				<executions>
-					<execution>
-						<phase>process-sources</phase>
-						<goals>
-							<goal>format</goal>
-							<goal>check</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-			<plugin>
-				<groupId>org.jacoco</groupId>
-				<artifactId>jacoco-maven-plugin</artifactId>
-				<version>0.8.5</version>
-				<executions>
-					<execution>
-						<goals>
-							<goal>prepare-agent</goal>
-						</goals>
-					</execution>
-					<!-- attached to Maven test phase -->
-					<execution>
-						<id>report</id>
-						<phase>test</phase>
-						<goals>
-							<goal>report</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-		</plugins>
-	</build>
+    <properties>
+        <java.version>11</java.version>
+        <version.javadoc.plugin>3.0.1</version.javadoc.plugin>
+        <apache.httpmime.version>4.5.9</apache.httpmime.version>
+        <apache.httpasyncclient.version>4.1.4</apache.httpasyncclient.version>
+        <lombok.version>1.18.10</lombok.version>
+        <nimbusds.version>7.9</nimbusds.version>
+        <eidas.psd2.sdk.version>1.27</eidas.psd2.sdk.version>
+        <mockito.core.version>3.0.0</mockito.core.version>
```

Comment:
I have formatted the file, sorry, the changes are:
- line 65: eidas version and lines 99, 100 and 101 to use the latest eidas library published in FR maven repo.
- line 73 to 77: Added module dependency
- lines from 146 to 168: Added FR maven repositories.